src/seedu/addressbook/parser/Parser.java

```diff
@@ -3,11 +3,7 @@
 import static seedu.addressbook.common.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
 import static seedu.addressbook.common.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
 
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.*;
```

Comment:
This currently violates the java coding standards: **Imported classes should always be listed explicitly.**

You can prevent IntelliJ from doing so by going through,
`Settings/Preferences -> Code Style -> Java -> Check "Use single class import"`
and
`Set Class count to use import with '*' & Names count to use static import with '*' to > 100`